contracts/pallet-ibc/docs/routing.md

```diff
@@ -175,7 +175,12 @@ impl<T: Config + Send + Sync> Module for IbcModule<T> {
         let success = "success".as_bytes().to_vec();
         let data = String::from_utf8(packet.data.clone()).ok();
         let packet = packet.clone();
```

Comment:
do you actually need to clone the packet? as far as I see, it's only taking a reference in line 182